### PR TITLE
 use eventbus instead PropertyChangeSupport to sync status & use ActionView.Builder to simplify action label view defining

### PR DESCRIPTION
**Pull request by wangmingliang-ms**

### CODE REVIEW

**Review by shenqianjin — APPROVED** [review]
LGTM

File: PluginsAndFeatures/azure-toolkit-for-intellij/.idea/inspectionProfiles/Project_Default.xml [hunk 1]
```diff
@@ -2,6 +2,7 @@
   <profile version="1.0" is_locked="true">
     <option name="myName" value="Project Default" />
     <inspection_tool class="AbsoluteAlignmentInUserInterface" enabled="false" level="WARNING" enabled_by_default="false" />
+    <inspection_tool class="AbstractBeanReferencesInspection" enabled="false" level="ERROR" enabled_by_default="false" />
```
Comment (Flanker32) [inline PluginsAndFeatures/azure-toolkit-for-intellij/.idea/inspectionProfiles/Project_Default.xml:5]: Please send another PR for inspection configuration
